### DISCUSSION TITLE
Renovate no longer automerges on Gitea / Forgejo v1.21.x

### DISCUSSION THREAD

**Discussion by viceice** [renovatebot/renovate#26246] [Request Help]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us what version of Renovate you run.

latest

### If you're self-hosting Renovate, select which platform you are using.

Gitea or Forgejo

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the problem

Because of a bug[^1] in Gitea / Forgejo renovate can't get the commit statuses of the branches, so automerge is no longer working.

There is another bug[^2] which prevents platform automerge.

[^1]: https://codeberg.org/forgejo/forgejo/issues/1929
[^2]: https://github.com/go-gitea/gitea/issues/24445

### Relevant debug logs

<details><summary>Logs</summary>

```

DEBUG: Processing existing PR (repository=forgejo-contrib/forgejo-renovate, branch=renovate/ghcr.io-visualon-renovate-37.x)
DEBUG: resolveBranchStatus(branchName=renovate/ghcr.io-visualon-renovate-37.x, ignoreTests=false) (repository=forgejo-contrib/forgejo-renovate, branch=renovate/ghcr.io-visualon-renovate-37.x)
DEBUG: GET https://code.forgejo.org/api/v1/repos/forgejo-contrib/forgejo-renovate/commits/renovate%2Fghcr.io-visualon-renovate-37.x/statuses = (code=ERR_NON_2XX_3XX_RESPONSE, statusCode=404 retryCount=0, duration=98) (repository=forgejo-contrib/forgejo-renovate, branch=renovate/ghcr.io-visualon-renovate-37.x)
DEBUG: Received 404 when checking branch status, assuming branch deletion (repository=forgejo-contrib/forgejo-renovate, branch=renovate/ghcr.io-visualon-renovate-37.x)
DEBUG: Passing error up (repository=forgejo-contrib/forgejo-renovate, branch=renovate/ghcr.io-visualon-renovate-37.x)
DEBUG: Passing PR error up (repository=forgejo-contrib/forgejo-renovate, branch=renovate/ghcr.io-visualon-renovate-37.x)
 INFO: Repository has changed during renovation - aborting (repository=forgejo-contrib/forgejo-renovate)
DEBUG: Repository result: repository-changed, status: onboarded, enabled: true, onboarded: true (repository=forgejo-contrib/forgejo-renovate)

```

</details>

### Have you created a minimal reproduction repository?

I have explained in the description why a minimal reproduction is impossible

**Comment by jamietanna** [comment]
Closing due to age, as I assume we've fixed this by now, @viceice ?
